frontend/src/components/common/ExportCSVButton.tsx

```diff
@@ -11,47 +11,66 @@ import {
   FormControl,
   Button,
   FormHelperText,
-  Flex,
+  FormErrorMessage,
   ScaleFade,
   Alert,
   AlertDescription,
   AlertIcon,
   Tooltip,
+  Grid,
+  GridItem,
+  Text,
 } from "@chakra-ui/react";
 import { TiExport } from "react-icons/ti";
-import { RangeDatepicker } from "chakra-dayzed-datepicker";
+import { SingleDatepicker } from "chakra-dayzed-datepicker";
 import CSVConverter from "../../helper/CSVConverter";
 import LogRecordAPIClient from "../../APIClients/LogRecordAPIClient";
-import { rangeDatePickerStyle } from "../../theme/forms/datePickerStyles";
+import { singleDatePickerStyle } from "../../theme/forms/datePickerStyles";
 
 const ExportCSVButton = (): React.ReactElement => {
   const [selectedDates, setSelectedDates] = useState<Date[]>([]);
+  const [startDate, setStartDate] = useState<Date | undefined>();
+  const [endDate, setEndDate] = useState<Date | undefined>();
+  const [dateError, setDateError] = useState<boolean>(false);
   const [showAlert, setShowAlert] = useState(false);
 
   const [isOpen, setOpen] = useState(false);
 
   const handleClear = () => {
-    setSelectedDates([]);
+    setStartDate(undefined);
+    setEndDate(undefined);
+    setDateError(false);
   };
 
   const handleOpen = () => {
     setOpen(true);
-    setSelectedDates([]);
+    handleClear();
   };
 
   const handleClose = () => {
     setOpen(false);
   };
 
-  const handleSubmit = async () => {
-    const dateRange = selectedDates.map((date) =>
-      date
+  const formatDate = (date: Date | undefined) => {
+    if (date !== undefined) {
+      return date
         .toLocaleString("fr-CA", { timeZone: "America/Toronto" })
-        .substring(0, 10),
-    );
+        .substring(0, 10);
+    }
+    return "";
```

Comment:
More correct to use `null` here instead of empty quotes since there's no date provided